### PR TITLE
Ada2 33

### PR DESCRIPTION
**Pull request by Archieduck**
Submit buttons disable for 1 second after pressed and form clears on submitting. For location report and region summary

### CODE REVIEW

File: AgentDiscoveries-Backend/src/main/resources/config.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Database properties
 database.url = jdbc:mysql://localhost:3306/agentdiscoveries
 database.username = root
-database.password = root
+database.password = password
```
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/resources/config.properties:5]: Do we want to be committing this? Can't remember what you guys agreed on so feel free to ignore

File: AgentDiscoveries-Frontend/app/src/components/submit-forms/submit-location-report.jsx [hunk 2]
```diff
@@ -130,6 +131,8 @@ export default class LocationReportSubmit extends React.Component {
 
     onSubmit(event) {
         event.preventDefault();
+        this.setState({ reportSubmitted : true });
+        setTimeout(() => this.setState({ reportSubmitted: false }), 1000); // button disabled for 5 seconds when report submitted
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/submit-forms/submit-location-report.jsx:135]: Nice, this looks good

**Review by RogVan — APPROVED** [review]
Looks really good, nice job

**Review by tim-barton96 — APPROVED** [review]
looks good

**Pull request merged**